src/main/java/github/scarsz/discordsrv/objects/threads/ChannelUpdater.java

```diff
@@ -0,0 +1,185 @@
+/*-
+ * LICENSE
+ * DiscordSRV
+ * -------------
+ * Copyright (C) 2016 - 2021 Austin "Scarsz" Shapiro
+ * -------------
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * END
+ */
+
+package github.scarsz.discordsrv.objects.threads;
+
+import alexh.weak.Dynamic;
+import github.scarsz.discordsrv.Debug;
+import github.scarsz.discordsrv.DiscordSRV;
+import github.scarsz.discordsrv.util.DiscordUtil;
+import github.scarsz.discordsrv.util.PlaceholderUtil;
+import github.scarsz.discordsrv.util.TimeUtil;
+import lombok.Getter;
+import net.dv8tion.jda.api.entities.GuildChannel;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.Bukkit;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+public class ChannelUpdater extends Thread {
+
+    @Getter private final Set<UpdaterChannel> updaterChannels = new HashSet<>();
+
+    public ChannelUpdater() {
+        setName("DiscordSRV - Channel Updater");
+    }
+
+    public void reload() {
+        // Deleting and recreating the list of updater channels
+        this.updaterChannels.clear();
+
+        final List<Map<?, ?>> configEntries = DiscordSRV.config().get("ChannelUpdater");
+
+        for (final Map<?, ?> configEntry : configEntries) {
+            Dynamic map = Dynamic.from(configEntry);
```

Comment:
Should be able to avoid doing `Dynamic#from` here by using `Dynamic#asList` / `Dynamic#children`